src/language/javascript.js

```diff
@@ -1,3 +1,5 @@
+const Rainbow = require('../../dist/rainbow.js');
```

Comment:
```suggestion
const Rainbow = require('./rainbow.js');
```

Same comment & concern as above.